user/languages/go.md

```diff
@@ -72,26 +81,23 @@ go_import_path: example.org/pkg/foo
 
 ## Dependency Management
 
-The default install step depends on the version of go:
-
-* if go version is greater than or equal to `1.2`
-
-  ```
-  go get -t -v ./...
-  ```
+The default `install` step of `travis_install_go_dependencies <go-version>
+[gobuild-args]` will behave differently depending on the Go version specified,
+as well as the presence of certain environment variables and file paths.
 
-* if go version is older than `1.2`
+If the Go version and environment are detected to support Go 1.11+ Modules or Go
+1.5+ vendoring, then we skip to Makefile detection. In all other cases, file
```

Comment:
Should `Makefile` be stylized?

Reply:
I'm good with either! I didn't in this case because we check for 4 different known file names.

Reply:
Ah, I see. I suppose we can leave it as is.